### PR TITLE
Consistency Cleanup

### PR DESCRIPTION
**Pull request by axchu**
### Fixes
- Fixes https://github.com/ethereum-optimism/roadmap/issues/281

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

**Comment by axchu** [comment]
The style we have decided to follow:
- for the Library contracts, `internal` functions will not have underscores, `private` functions will.  There are no public functions.
- for the OVM contracts, `public` functions will not have underscores, `internal` and `private` functions will.

File: contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol [hunk 1]
```diff
@@ -249,4 +215,43 @@ contract OVM_FraudVerifier is Lib_AddressResolver, OVM_FraudContributor, iOVM_Fr
...
         return address(getStateTransitioner(_preStateRoot, _txHash)) != address(0);
     }
+
+
+    /************************************
+     * Private Functions: Verification *
+     ************************************/
+
+    // NB: Stack too deep :/
+    function _deployTransitioner(bytes32 _preStateRoot, bytes32 _txHash) private {
```
Comment (kfichter) [inline contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol:225]: Would you mind just changing the structure of this signature to match the other signatures in this file (with each parameter on its own line, and `private` on its own line too) + add a function comment here? Thank you!

File: contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol [hunk 2]
```diff
@@ -249,4 +215,43 @@ contract OVM_FraudVerifier is Lib_AddressResolver, OVM_FraudContributor, iOVM_Fr
...
+            resolve("OVM_StateTransitionerFactory")
+        ).create(
+            address(libAddressManager),
+            _preStateRoot,
+            _txHash
+        );
+    }
+
+    // NB: Stack too deep :/
+    function _cancelStateTransition(
```
Comment (kfichter) [inline contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol:236]: Same as above with fixing signature + adding a function comment.

**Review by kfichter — CHANGES_REQUESTED** [review]
Looks good, requested a few minor changes.

File: contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol [hunk 3]
```diff
@@ -249,4 +215,60 @@ contract OVM_FraudVerifier is Lib_AddressResolver, OVM_FraudContributor, iOVM_Fr
...
+    /************************************
+     * Private Functions: Verification *
+     ************************************/
+
+    /**
+     * Deploys a transitioner.
+     * @param _preStateRoot State root before the fraudulent transaction.
+     * @param _txHash The transaction for the state root.
+     */
+    // NB: Stack too deep :/
```
Comment (kfichter) [inline contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol:229]: We can get rid of the "stack too deep" comments here

**Review by kfichter — APPROVED** [review]
LGTM! Approving but let's wait to merge until we're sure that we want to make updates to master.

**Comment by ben-chain** [comment]
This does not change any public interfaces, should be good to merge.

**Pull request closed without merging**